doc/release-notes-17437.md

```diff
@@ -1,5 +1,5 @@
 Low-level RPC Changes
 ===
 
-- The RPC gettransaction, listtransactions and listsinceblock responses now also
-includes the height of the block that contains the wallet transaction, if any.
+- The RPC gettransaction, listtransactions, and listsinceblock responses now
```

Comment:
4e206a89383eaca83acbbf9533e4aec62fe9fee7

Comma before "and" is not needed here? TIL both ways are correct but do you think this is more correct?

Reply:
I grew up being taught to not use the Oxford comma, and it could be considered less aesthetic, but it seems to have made a resurgence. It has the benefit of reducing potential confusion in writing that needs to be clear. Without it, meanings of sentences can change. So it can be a good idea to use it for documentation writing. That said, I don't feel strongly about it here :)

https://www.rd.com/culture/oxford-comma-proper-use/
https://thewritelife.com/is-the-oxford-comma-necessary/
https://thewritepractice.com/why-you-need-to-be-using-oxford-commas/
https://en.wikipedia.org/wiki/Serial_comma